pulsar-client-shaded/pom.xml

```diff
@@ -74,6 +74,7 @@
                   <include>com.google.guava:guava</include>
                   <include>com.google.code.gson:gson</include>
                   <include>com.fasterxml.jackson.core</include>
+                  <include>com.fasterxml.jackson.module</include>
```

Comment:
The same should also be done for `pulsar-client-admin-shaded` and for kafka wrapper too. Unfortunately we haven't yet found a way to have the shading conf in a single place